Explorer/Assets/DCL/Character/CharacterCamera/Systems/ControlCinemachineVirtualCameraSystem.cs

```diff
@@ -22,8 +22,9 @@ namespace DCL.Character.CharacterCamera.Systems
     [UpdateInGroup(typeof(CameraGroup))]
     public partial class ControlCinemachineVirtualCameraSystem : BaseUnityLoopSystem
     {
+        private readonly ICinemachineCameraAudioSettings cinemachineCameraAudioSettings;
+
```

Comment:
wait, this is again just application of the code style. I don't share your fear of polluting history, since with modern IDE it is quite easy to manage